app.rb

```diff
@@ -58,11 +69,11 @@ def auth_server_uri
     puts "Logging in via GET"
     request = OneLogin::RubySaml::Authrequest.new
     puts "Request: #{request}"
-    ial = params[:ial] || '1'
-    aal = params[:aal] || '2'
-    skip_encryption = params[:skip_encryption]
+    ial = get_param(:ial, ['1', '2', '2-strict', '0']) || '1'
+    aal = get_param(:aal, ['1', '2', '3', '3-hspd12']) || '1'
```

Comment:
The default was `2` before:

```suggestion
    aal = get_param(:aal, ['1', '2', '3', '3-hspd12']) || '2'
```

Reply:
whoops no, meant to leave the default 🤦  this is what I get for rushing